lms/static/sass/base/_variables.scss

```diff
@@ -201,7 +201,7 @@ $shadow-d2: rgba($black, 0.6) !default;
 // system feedback-based colors
 $error-color: rgb(253, 87, 87) !default;
 $warning-color: rgb(181,42,103) !default;
-$confirm-color: rgb(0, 136, 1) !default;
+$confirm-color: rgb(0, 132, 1) !default;
```

Comment:
$confirm-color is only used by the instructor dashboard CSS, and only a minor tweak was needed to get sufficient contrast.